packages/@react-spectrum/provider/docs/Provider.mdx

```diff
@@ -32,42 +30,38 @@ import themeLight from '@adobe/spectrum-css-temp/vars/spectrum-light-unique.css'
 </Provider>
 ```
 
-## Children Props
+## Global Settings
 
-As a convenience, Provider supports passing common props to all components
-within it. For example, an entire group of components could be disabled by
-setting `isDisabled` in one place rather than on each individual element.
-These props are `isDisabled`, `isEmphasized`, `isQuiet`, `isReadOnly`,
-`isRequired`, and `validationState`.
+Props such as `theme`, `colorScheme`, `locale`, and `scale` are applied to all
+children components, included nested Providers. The `theme` allows the
+definition of custom CSS variables for the `colorScheme` and `scale` prop values.
 
-### Quiet
+### Color Scheme and Scale
 
 ```tsx example
-<Provider isQuiet>
-  <Form>
+<Provider colorScheme="dark" scale="large">
+  <Form margin="15px">
     <div>
       <TextField label="Email" />
     </div>
     <Checkbox>
       Checkbox Label
     </Checkbox>
-    <div>
+    <div style={{'margin-bottom': '15px'}}>
       <Button variant="primary">Button</Button>
     </div>
   </Form>
 </Provider>
 ```
 
-## Global Settings
+### Theme
 
-Props such as `theme`, `colorScheme`, `locale`, and `scale` are applied to all
-children components, included nested Providers. The `theme` allows the
-definition of custom CSS variables for the `colorScheme` and `scale` prop values.
+The Providers theme is the CSS variables for scale and color scheme values.
 
-### Color Scheme and Scale
+```tsx
+import {theme} from '@react-spectrum/theme-default';
```

Comment:
Is this mean to be a `tsx example` or is only having `tsx` here intentional?

Reply:
it is not meant to be a `tsx example`

Reply:
I don't understand this block
First, why isn't it an example?
Second: ```The Provider's `theme` is the CSS variables for `colorScheme` and `scale` values.``` why are colorScheme and scale mentioned in relation to theme when it only shows `theme` in this block of tsx

Reply:
I had theme as an example which Danni didn't like. She suggested this example and we agreed it should just be tsx and not a tsx example. She also identified that theming needs to have its own docs page.

That sentence was meant to encapsulate the logic of https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-spectrum/provider/src/Provider.tsx#L44-L45 and https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-spectrum/provider/src/Provider.tsx#L136-L137

I ran it by Danni, but looking at the chat I'm not sure if she okayed it or a different conversation. Do you have a better suggestion of explaining the relationship between these three props?

Reply:
Ok, I'm cool with it not being an example in favor of a theming docs page.

To explain the relationship between those props. I'd really focus more on the tsx and add the two props that are mentioned in the description text but missing from the tsx.